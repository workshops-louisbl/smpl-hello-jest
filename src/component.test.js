import React from 'react';
import TestRenderer from "react-test-renderer";
import ReactComponent from "./component";

test("render react component", () => {
  const component = TestRenderer.create(
    <ReactComponent name="Georges" />
  );

  expect(component).toMatchSnapshot();
})
