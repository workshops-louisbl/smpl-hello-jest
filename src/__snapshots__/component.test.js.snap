// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render react component 1`] = `
<div>
  <h1>
    Georges
  </h1>
</div>
`;
